.azure/pipelines/ci.yml

```diff
@@ -697,7 +697,7 @@ stages:
           # Just uploading artifacts/logs/ files can take 15 minutes. Doubling the cancel timeout for this job.
           cancelTimeoutInMinutes: 30
           buildArgs: -all -pack -test -binaryLog /p:SkipHelixReadyTests=true /p:SkipIISNewHandlerTests=true /p:SkipIISTests=true
-                     /p:SkipIISExpressTests=true /p:SkipIISNewShimTests=true /p:RunTemplateTests=false
+                     /p:SkipIISExpressTests=true /p:SkipIISNewShimTests=true /p:RunTemplateTests=false /p:RunBlazorPlaywrightTemplateTests=true
```

Comment:
Will setting this new variable (RunBlazorPlaywrightTemplateTests) to `false` result in the current behavior (as if we don't have this change in place)?

Reply:
It's fine to just omit `RunBlazorPlaywrightTemplateTests` in that case as it'll default to using the value of `RunTemplateTests`, but yes setting it to `false` would work as well.

https://github.com/dotnet/aspnetcore/blob/20ebf882bf561130d3288a419ee2caa6f3d343c4/src/ProjectTemplates/test/Templates.Blazor.Tests/Templates.Blazor.Tests.csproj#L10-L11